dyno-queues-redis/src/main/java/com/netflix/dyno/queues/demo/DynoQueueDemo.java

```diff
@@ -0,0 +1,81 @@
+package com.netflix.dyno.queues.demo;
+
+import com.netflix.dyno.demo.redis.DynoJedisDemo;
+import com.netflix.dyno.jedis.DynoJedisClient;
+import com.netflix.dyno.queues.DynoQueue;
+import com.netflix.dyno.queues.Message;
+import com.netflix.dyno.queues.redis.v2.QueueBuilder;
+import com.netflix.dyno.queues.shard.DynoShardSupplier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+public class DynoQueueDemo extends DynoJedisDemo {
+
+    private static final Logger logger = LoggerFactory.getLogger(DynoQueue.class);
+    public DynoQueueDemo(String clusterName, String localRack) {
+        super(clusterName, localRack);
+    }
+
+    public DynoQueueDemo(String primaryCluster, String shadowCluster, String localRack) {
+        super(primaryCluster, shadowCluster, localRack);
+    }
+
+    public static void main(String[] args) throws IOException {
+        final String clusterName = args[0];
```

Comment:
Is there a documentation/comment or some exception so that we can notify the user that the argument is not a proper cluster name?

Reply:
i'll add a comment on the function to document this.